docs/gpt.rst

```diff
@@ -0,0 +1,67 @@
+gpt
+=======
+
+.. dfhack-tool::
+    :summary: AI-generated written content!
+    :tags: fort gameplay
+
+Enables a UI for submitting knowledge item descriptions to OpenAI for generating
+poetry, star charts, and excerpts from longer works such as biographies, dictionaries,
+treatises on technological evolution, comparative biographies, cultural histories,
+autobiographies, cultural comparisons, essays, guides, manuals, and more.
+
+``enable gpt``
+=======
+Enables the plugin. The overlay will be shown when a knowledge item or unit view sheet is open.
+
+``disable gpt``
+=======
+Disables the plugin.
+
+Setup:
+
+1. Register for an OpenAI API account. It must be a paid or active trial account.
+2. Generate an API token for your account.
+3. Save your OpenAI API token to a file at the root of your DF directory, `oaak.txt`.
```

Comment:
we try not to put any user data in the DF root directory. I suggest putting it in `dfhack-config/`.

Reply:
OK, makes sense